sdks/python/apache_beam/transforms/external_test.py

```diff
@@ -199,6 +199,7 @@ def test_pipeline_generation_with_runner_overrides(self):
         '--dry_run=True',
         '--sdk_location=container',
         '--runner=DataflowRunner',
+        '--experiments=use_runner_v2',
```

Comment:
Is this necessary for only testing pipeline generation? If so, do we also need `use_portable_job_submission` ?

Reply:
Removed this.